src/connectionTool/utills/Verifier.java

```diff
@@ -0,0 +1,33 @@
+package connectionTool.utills;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Verifier {
+
+    private static <T> void verify(List<String> requiredArguments, T t, String... args) throws NoSuchFieldException, IllegalAccessException {
+        for (String arg : args) {
+            Field field  = t.getClass().getDeclaredField(arg);
```

Comment:
I think we don't have to use reflection, we could model data to make it clear what verifier expect, what fields are required.
We could take a look together tomorrow/on Friday if you'd like.